boefjes/boefjes/job_handler.py

```diff
@@ -33,146 +26,73 @@
 )
 
 
-def get_octopoes_api_connector(org_code: str) -> OctopoesAPIConnector:
-    return OctopoesAPIConnector(str(settings.octopoes_api), org_code, timeout=settings.outgoing_request_timeout)
-
-
-@cache
-def boefje_env_variables() -> dict:
-    """
-    Return all environment variables that start with BOEFJE_. The returned
-    keys have the BOEFJE_ prefix removed.
-    """
-
-    boefje_variables = {}
-    for key, value in os.environ.items():
-        if key.startswith("BOEFJE_"):
-            boefje_variables[key.removeprefix("BOEFJE_")] = value
-
-    return boefje_variables
-
+class DockerBoefjeHandler(Handler):
+    CACHE_VOLUME_NAME = "openkat_cache"
```

Comment:
Since we have multiple workers, we can have multiple boefjes, and possibly multiple of the same boefjes, using the same volume. This Volume is not shareable I'd guess. So have we tested this?

Reply:
I've tested this a while ago with RPKI, which seemed to work. The same volume is attached to each boefje run, hence it works on the same machine at least.